libs/blocks/quiz/quiz.css

```diff
@@ -873,8 +610,366 @@ TODO: Need to check if we can skip this somehow.
   top: 6px;
   margin-left: 20px;
 }
-@media screen and (min-width: 900px) {
-  .dot-indicators:last-of-type {
-    margin-top: var(--spacing-xl);
+
+@media screen and (min-width: 600px) {
+  .quiz-container .action-area {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    gap: var(--spacing-xs);
+  }
+
+  .quiz-container .background .mobileOnly,
+  .quiz-container .background .desktopOnly {
+    display: none;
+  }
+
+  .quiz-container .background .tabletOnly {
+    display: block;
+  }
+
+  /* Split */
+  .quiz-container.split {
+    display: flex;
+    justify-content: center;
+  }
+
+  .quiz-container.split .action-area {
+    gap: var(--spacing-s);
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .quiz-container.split .foreground.container {
+    flex-direction: row;
+    padding: var(--spacing-xl) 0;
+  }
+
+  .quiz-container.split .foreground.container .text {
+    max-width: calc(50% - var(--grid-column-width));
+  }
+
+  .quiz-container.split .media.bleed {
+    position: absolute;
+    width: 50vw;
+    right: 0;
+    height: 100%;
+  }
+
+  .quiz-container.split:not(.row-reversed) .media-credit {
+    text-align: right;
+  }
+
+  .quiz-container.split .text .icon-area img {
+    max-height: 56px;
+    max-width: 212px;
+  }
+
+  .quiz-container.split.large .text .icon-area img {
+    max-height: 80px;
+  }
+
+  .quiz-container.split.row-reversed .media.bleed {
+    right: auto;
+    left: 0;
+  }
+
+  .quiz-container.split.row-reversed .foreground.container {
+    justify-content: flex-end;
+  }
+
+  .quiz-container.split .media.bleed picture,
+  .quiz-container.split .media.bleed video {
+    height: 100%;
+    object-fit: fill;
+  }
+
+  .quiz-container.split.large .text {
+    margin: 0;
+  }
+
+  .quiz-container.split.small .media img,
+  .quiz-container.split.large .media img,
+  .quiz-container.split .media img,
+  .quiz-container.split.small .media video,
+  .quiz-container.split.large .media video,
+  .quiz-container.split .media video {
+    max-height: initial;
+  }
+
+  .quiz-container.split .button-justified-mobile {
+    width: auto;
+  }
+}
+
+@media screen and (min-width: 899px) and (max-width: 1221px) {
+  .quiz-page .gnav {
+    padding: 0 20px;
+  }
+}
+
+@media screen and (max-width: 899px) {
```

Comment:
This is the only place where you're using `max-width`. Why not move this declaration to L374 and then, if needed, counteract it on L722, in the `@media screen and (min-width: 900px)` section. Mixing mobile-first and desktop-first approach can be detrimental to code maintainability in the future, it's better to follow a pattern.